tests/mbis-6/input.dat

```diff
@@ -61,8 +61,9 @@ set {
   max_radial_moment 4
 }
 
+set scf_properties ['MBIS_CHARGES', 'MBIS_VOLUME_RATIOS']
```

Comment:
This route (set scf_properties, then energy()) should run now, right? ref.dat has error at end, so I may have missed the point.

Reply:
Yes, I copied the wrong output